src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/overlays/mediacropdetails.controller.js

```diff
@@ -56,4 +58,14 @@
                 }
             }
 
+            function cropNotSet() {
+                var model = $scope.model;
+                return !(model.cropSize || {}).width && model.target.thumbnail;
+            }
+
+            function cropSet() {
+                var model = $scope.model;
+                return (model.cropSize || {}).width && model.target.thumbnail;
+            }
+
```

Comment:
Do we need two functions for this? Can't er just set a boolean scope property?
E.g. `vm.cropSet`, `vm.cropsExists` or similar?

Reply:
Not really, but then again "cropNotSet" is easier on the eye than "!cropSet". Also, if going on the path mentioned in the other comment, one of the two might get renamed, refactored and extended individually.

Reply:
Sure, but we could use same function which return true or false.
You can then use two different scope properties or explicit specify the value, e.g. "cropSet === true" and "cropSet === false".